pkg/common/db/database/admin.go

```diff
@@ -121,6 +125,21 @@ func (o *AdminDatabase) UpdateAdmin(ctx context.Context, userID string, update m
 	return o.admin.Update(ctx, userID, update)
 }
 
+func (o *AdminDatabase) ChangePassword(ctx context.Context, userID string, newPassword string) error {
```

Comment:
It would be greatly appreciated if you could kindly assist in adding testing to the interface. This is currently on our todo list. Thank you in advance for your help!

Reply:
ok